python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -304,6 +304,110 @@ def __toonSummary( plug ) :
 
 		],
 
+		"attributes.cameraAutobumpVisibility" : [
+
+			"description",
+			"""
+			Whether or not the autobump is visible to camera
+			rays.
+			""",
+
+			"layout:section", "Visibility",
```

Comment:
Autobump visibility is too esoteric to allow it to clutter up the standard visibility section. If we put it in its own section we could also omit the "autobump" part from the labels, so they would fit in the NodeEditor without getting clipped.

I wonder though, is ArnoldAttributes the right place to put these options? The `autobump` setting itself is on the ArnoldDisplacement node, so perhaps that is where the visibility attributes belong? Do you have any opinion on that @danieldresser-ie?